.github/workflows/scripts/check_bundle_size.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  checkBaseBundleSize: () => {
+    const fs = require('fs');
+    const jsonString = fs.readFileSync('.github/analyze/bundles.json');
```

Comment:
Sorry if I missed it somewhere, but is this file getting generated from a step or something like that?

Reply:
Ah nvm, I see it now, you make the build script run with the `ANALYZE` env var, sorry about that!